packages/react-composites/src/composites/CallComposite/CallComposite.tsx

```diff
@@ -102,6 +102,19 @@ export interface RemoteVideoTileMenuOptions {
    */
   isHidden?: boolean;
 }
+
+/* @conditional-compile-remove(click-to-call) */
+/**
+ * Options for the local video tile in the Call composite.
+ *
+ * @beta
+ */
+export interface LocalVideoTileOptions {
+  /**
+   * Position of the local video tile. If unset will render the local tile in the floating local position.
+   */
+  position?: 'grid' | 'floating' | 'hidden';
```

Comment:
Can we have comments explicit on what grid and floating mean? 
Position wise, where is it, where does grid place it. 
If grid is default layout, should we just name it `default`?

Reply:
I think mentioning it in the comments where it will land is correct, but for the composite default is 'floating' so we can't use that terminology here.